internal/app/machined/pkg/runtime/v1alpha1/bootloader/grub/grub.go

```diff
@@ -107,6 +108,8 @@ func (g *Grub) Labels() (current, next string, err error) {
 		return "", "", fmt.Errorf("unknown grub menuentry: %q", current)
 	}
 
+	next = fmt.Printf("%s %s", next, version.Tag)
```

Comment:
feels this would break our detection of labels via BootA/BootB

Reply:
Yeah I am sure this is going to break something right now. Kind of working through it and leaning on CI right now since my dev environment is broken at the moment.